manifest.json

```diff
@@ -7,7 +7,10 @@
             "programs": [
                 {
                     "id": "run-vehicledatabroker-cli",
-                    "executable": "./runtime_local/src/run-vehicledatabroker-cli.sh"
+                    "executable": "python3",
+                    "args": [
+                        "./runtime_local/src/run-vehicledatabroker-cli.py"
+                    ]
```

Comment:
if https://github.com/eclipse-velocitas/cli/pull/291 is fine you could add "interactive": true

Reply:
Done